pkg/api/types.go

```diff
@@ -1595,6 +1595,45 @@ type Handler struct {
 	TCPSocket *TCPSocketAction
 }
 
+// DeleteExecAction describes a "run in container" action that will be invoked
```

Comment:
The `Delete` indicates it's an `ExecAction` plus a deletion reason string. Made it clear in the comment.